### PR TITLE
Fix Strings support

### PR DESCRIPTION
**Pull request by hector6872**
Fix Strings support #12 
Add tests

### CODE REVIEW

File: sqlite/src/com/nivabit/kuery/sqlite/SQLiteDialect.kt [hunk 1]
```diff
@@ -489,7 +490,8 @@ object SQLiteDialect : Dialect {
     }
 
     private fun appendValue(builder: StringBuilder, value: Any?) {
-        if (value == null) builder.append("NULL")
-        else builder.append(value)
+        value?.let {
+            builder.append(if (value is String) "\'$value\'" else value)
```
Comment (x2bool) [inline sqlite/src/com/nivabit/kuery/sqlite/SQLiteDialect.kt]: So if the value contains a single quote character, the output would be invalid SQL. A string value should be escaped.

https://stackoverflow.com/questions/603572/escape-single-quote-character-for-use-in-an-sqlite-query

Comment (hector6872) [reply sqlite/src/com/nivabit/kuery/sqlite/SQLiteDialect.kt]: Not anymore :)

**Comment by x2bool** [comment]
Looks good. I am going to need some time to test it.

**Comment by x2bool** [comment]
Thank you!

**Pull request merged**